lib/ts/Arena.h

```diff
@@ -21,12 +21,13 @@
   limitations under the License.
  */
 
-#ifndef __ARENA_H__
-#define __ARENA_H__
+#pragma once
```

Comment:
That's fair. We do want to make the code better. I usually think of that as fixing a loop structure because it's poor when I am just changing some behavior inside of it. This seems more like something we should decide to do as a coding standard and make the changes to all header files as they should be consistent.